src/DynamoCoreWpf/Views/Core/AnnotationView.xaml.cs

```diff
@@ -237,7 +237,10 @@ private void GroupTextBox_OnTextChanged(object sender, TextChangedEventArgs e)
 
             SetTextMaxWidth();
             SetTextHeight();
-            ViewModel.WorkspaceViewModel.HasUnsavedChanges = true;
+            if (GroupTextBox.ActualHeight > 0 && GroupTextBox.ActualWidth > 0)
```

Comment:
if the textbox has custom group name and description already, won't the actual H & W be greater than 0. We don't want to set the HasUnsavedChanges to true in that case.

Reply:
Yes, at-least at initialization, when WPF is creating groups.

The group text is bind to AnnotationText, which is set to the stored value when the control is initialized, that action should not trigger HasUnsaved changes, because we are initializing the group for the first time. 
I tried capturing this by IsLoaded and IsReady events, but at this point none of them worked, so I needed a way to differentiate between an explicit action by the user to change the text versus group text initialization on workspace load. So I found that H & W are 0 at this point which won't be the case when any user action would trigger it.